ADMINS.md

```diff
@@ -0,0 +1,9 @@
+## Admins
+| Admin             | GitHub ID                                         | Affiliation |
+|------------------------|---------------------------------------------------|-------------|
+| Anantha Krishna Bhatta | [akbhatta](https://github.com/akbhatta)           | Amazon      |
+| David Cui              | [davidcui-amzn](https://github.com/davidcui-amzn) | Amazon      |
+| Joshua Li              | [joshuali925](https://github.com/joshuali925)     | Amazon      |
+| Zhongnan Su            | [zhongnansu](https://github.com/zhongnansu)       | Amazon      |
```

Comment:
I believe admins are different from maintainers, currently we only have one admin https://github.com/opensearch-project/trace-analytics/blob/2aec3399bea4f07c2add94912a4b36da930efab8/ADMINS.md

Reply:
can we apply for admin access only for certain repo?